src/ui/components/Header/Header.tsx

```diff
@@ -62,7 +70,7 @@ function Links({ recordingTarget }: { recordingTarget: RecordingTarget | null })
       {showShareButton ? <ShareButton /> : null}
       <Avatars recordingId={recordingId} />
       {showViewToggle && <ViewToggle />}
-      <UserOptions />
+      <UserOptions toggleReplayAssist={toggleReplayAssist} />
```

Comment:
`UserOptions` isn't using this new prop. Remove it (I think we can revert all of the changes in this file?